### PR TITLE
[IMP] Website_crm : admin not set as follower, website sale set as sales...

### PR DESCRIPTION
**Pull request by tivisse**
... team
TASK : https://www.odoo.com/web?db=openerp#id=11483&view_type=form&model=project.task
PAD : https://pad.odoo.com/p/r.72WvL363pMQB8Vmv

### CODE REVIEW

File: addons/website_crm/controllers/main.py [hunk 1]
```diff
@@ -77,7 +78,6 @@ def dict_to_str(title, dictvar):
             return request.website.render(kwargs.get("view_from", "website.contactus"), values)
 
         try:
-            values['medium_id'] = request.registry['ir.model.data'].get_object_reference(request.cr, SUPERUSER_ID, 'crm', 'crm_tracking_medium_website')[1]
```
Comment (tde-banana-odoo) [inline addons/website_crm/controllers/main.py]: Why removing that ?

Comment (tde-banana-odoo) [inline addons/website_crm/controllers/main.py]: I think

 - replace crm_tracking_medium_website by crm_medium_website that exists
 - change get_object_refernece by xmlid_to_res_id, that returns False if the object does not exist anymore
 - remove the try/catch, values will be False / False if nothing exist

This should solve several issues...

**Comment by tivisse** [comment]
@tde-banana-odoo Changes done. Thxx

**Comment by tde-banana-odoo** [comment]
Looks good to me.

**Comment by tde-banana-odoo** [comment]
Forward port note: crm_medium_website has been moved to utm module (is now utm.utm_medium_website.

**Pull request merged**